deployments/README.md

```diff
@@ -107,7 +107,7 @@ dashboard can be accessed at `http://localhost:9090`. The Grafana dashboard can
 
 Here is an example `.env` file which is used for production deployments and network updates. For
 security reasons it is not kept as part of version control. When deploying the network this
-file should be correctly populated and kept in the [`rialto`](./rialto) folder.
+file should be correctly populated and kept in the [`rialto`](./rialto) folder. The `UI_*_PROVIDER` variables let you define the urls of the nodes that the user interface will connect to. Per default, it will connect to the local substrate and ethereum nodes, using their default RPC port. UI_EXPECTED_ETHEREUM_NETWORK_ID should be set to the ethereum network id. This is used by the user interface to prevent users from connecting their Metamask account to another network.
```

Comment:
Lines should be less than 100 characters in length. Maybe this is something we should enforce in our editor config actually...